numba/annotations/template.html

```diff
@@ -0,0 +1,221 @@
+<html>
+
+<head>
+
+<style>
+
+.annotation_table {
+color: #000000;
+font-family: monospace;
+margin: 5px;
+}
+
+.object_tag {
+color: #cc3300;
+font-weight: bold;
+}
+
+.lifted_tag {
+color: #00cc33;
+font-weight: bold;
+}
+
+.llvm_code {
+color: grey;
+font-style: italic;
+}
+
+.metadata {
+border-bottom: medium solid black;
+display: inline-block;
+padding: 5px;
+}
+
+.annotations {
+padding: 5px;
+}
+
+.hidden {
+display: none;
+}
+
+.buttons {
+padding: 10px;
+cursor: pointer;
+}
+
+</style>
+
+<script type="text/javascript">
+
+window.onload = function() {
+
+%for func_id, func_key in enumerate(func_data):
+
+    <%func_id_str = str(func_id)%>
+    document.getElementById('func{{func_id_str}}_toggle_llvm').onclick = function() {
+        if (this.innerHTML == 'show llvm') {
+            var llvm_lines = getElementsByClassName('func{{func_id_str}}_llvm');
+            for (var i = 0; i < llvm_lines.length; i++) {
+                llvm_lines[i].setAttribute('class', 'llvm_code func{{func_id_str}}_llvm');
+            }
+            this.innerHTML = 'hide llvm'
+        }
+        else {
+            var llvm_lines = getElementsByClassName('func{{func_id_str}}_llvm');
+            for (var i = 0; i < llvm_lines.length; i++) {
+                llvm_lines[i].setAttribute('class', 'llvm_code func{{func_id_str}}_llvm hidden');
+            }
+            this.innerHTML = 'show llvm'
+        }
+    }
+
+    %endfor
+
+}
+
+/*
+    Alternate getElementsByClassName function that works on older versions
+    of Internet Explorer.
+	Developed by Robert Nyman, http://www.robertnyman.com
+	Code/licensing: http://code.google.com/p/getelementsbyclassname/
+*/	
+var getElementsByClassName = function (className, tag, elm){
```

Comment:
Do we want to use jQuery instead? The main advantage to this is that's it's lightweight, but I'm not sure that's really important. For jQuery we would have two possibilities:
- bundle it
- reference a well-known jQuery URL in a CDN (there are official ones)